config/opendatahub-io/org.yaml

```diff
@@ -313,6 +319,7 @@ orgs:
       repo-odh-model-controller-write:
         description: "Members with Write access to the odh-model-controller repository"
         members:
+        - amadhusu
```

Comment:
You don't need this access. I'm guessing you copied Humair's permissions, and his access here is an old artifact of a previous project he worked on.